include/lbann/trainers/trainer.hpp

```diff
@@ -66,7 +65,9 @@ class trainer {
   /** Archive for checkpoint and restart */
   template <class Archive> void serialize(Archive & ar) {
     ar(CEREAL_NVP(m_persist),
-       CEREAL_NVP(m_max_mini_batch_size));
+       CEREAL_NVP(m_max_mini_batch_size),
+       CEREAL_NVP(m_random_seed),
+       CEREAL_NVP(m_data_seq_random_seed));
```

Comment:
Does the checkpoint need access to the original seeds? Aren't the current seed values the important thing for reproducibility?

Reply:
Some of the seeds are set by the command line arguments so if a new execution doesn't have all of the right flags, they would be different.

Reply:
But once the model starts training, the RNG seeds are going to change as the generator is used. At that point, the original values and their origins don't matter anymore for restarting.